homeassistant/components/adax/climate.py

```diff
@@ -0,0 +1,157 @@
+"""Support for Adax wifi-enabled home heaters."""
+import logging
+
+from adax import Adax
+import voluptuous as vol
+
+from homeassistant.components.climate import PLATFORM_SCHEMA, ClimateEntity
+from homeassistant.components.climate.const import (
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    CONF_PASSWORD,
+    PRECISION_WHOLE,
+    TEMP_CELSIUS,
+)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import ACCOUNT_ID
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {vol.Required(ACCOUNT_ID): cv.string, vol.Required(CONF_PASSWORD): cv.string}
+)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
Does this make sense to setup a yaml config for a new integration?

Reply:
Leftover from the  HACS integration :/
Will fix